src/media/js/site/components/header.js

```diff
@@ -39,24 +40,59 @@ class HeaderLogin extends React.Component {
     loginHandler: React.PropTypes.func.isRequired,
     logoutHandler: React.PropTypes.func.isRequired,
     isLoggedIn: React.PropTypes.bool.isRequired,
+    showingUserDropdown: React.PropTypes.bool.isRequired,
+    toggleUserDropdown: React.PropTypes.func.isRequired,
   };
+
+  renderAnonNav() {
+    return (
+      <nav className="header--nav">
+        <ul>
+          <li><LoginButton isSignup={true} {...this.props}/></li>
+          <li><LoginButton {...this.props}/></li>
+        </ul>
+      </nav>
+    );
+  }
+
+  handleLogoutClick = evt => {
```

Comment:
i'd move this up so the two renders are next to each other, and i've been using ```e``` over ```evt``` but not a big deal